index.Rmd

```diff
@@ -6,6 +6,7 @@ bibliography: [book.bib]
 biblio-style: apalike
 link-citations: yes
 github-repo: microbiome/OMA
+always_allow_html: true
 ---
```

Comment:
Fixes

![image](https://user-images.githubusercontent.com/60338854/152405762-698a01af-693a-41a4-a040-4283a51217ea.png)

Reply:
Removed since it did not work